index.html

```diff
@@ -97,53 +102,24 @@ <h2>Multi-Post Stories</h2>
               </ul>
                       <button class="see-project" type="button">See Project</button>
             </div>
-        </div>
-      
-      <div class="tonic-card">

```

Comment:
### Your Project
![white-space (copy)](https://user-images.githubusercontent.com/93039359/179303008-a57b66da-9b96-493f-88d2-af5701026c29.png)


- [ ] You have a white spacing on the left side and because of that, you have a horizontal scrollbar. Please remove it as requested by the previous reviewer. :+1: